apps/assessment_registry/dashboard_schema.py

```diff
@@ -0,0 +1,335 @@
+
+
+class AssessmentPerSectorAndGeoAreaCountByDateType(graphene.ObjectType):
+    count = graphene.Int(required=True)
+    sector = graphene.Field(AssessmentRegistrySectorTypeEnum)
+    locations = graphene.Int(required=False)
+
+
+class AssessmentByLeadOrganizationCountByDateType(AssessmentCountByDateType):
+    organization = graphene.Int(required=False)
+
+
+class AssessmentDashboardStatisticsType(graphene.ObjectType):
+    total_assessment = graphene.Int(required=True)
+    total_stakeholder = graphene.Int(required=True)
+    total_collection_technique = graphene.Int(required=True)
+    assessment_count = graphene.List(AssessmentCountType)
+    stakeholder_count = graphene.List(StakeholderCountType)
+    collection_technique_count = graphene.List(CollectionTechniqueCountType)
+    total_multisector_assessment = graphene.Int(required=True)
+    total_singlesector_assessment = graphene.Int(required=True)
+    assessment_geographic_areas = graphene.List(AssessmentDashboardGeographicalArea)
+    assessment_by_over_time = graphene.List(AssessmentCountByDateType)
+    assessment_per_framework_piller = graphene.List(AssessmentFocusCountByDateType)
+    assessment_per_affected_group = graphene.List(AssessmentAffectedGroupCountByDateType)
+    assessment_per_humanitarian_sector = graphene.List(AssessmentHumanitrainSectorCountByDateType)
+    assessment_per_protection_management = graphene.List(AssessmentProtectionInformationCountByDateType)
+    assessment_per_affected_group_and_sector = graphene.List(AssessmentPerAffectedGroupAndSectorCountByDateType)
+    assessment_per_affected_group_and_geoarea = graphene.List(AssessmentPerAffectedGroupAndGeoAreaCountByDateType)
+    assessment_per_sector_and_geoarea = graphene.List(AssessmentPerSectorAndGeoAreaCountByDateType)
+    assessment_by_lead_organization = graphene.List(AssessmentByLeadOrganizationCountByDateType)
+
+    @staticmethod
+    def custom_resolver(root, info, _filter):
+        assessment_qs = AssessmentRegistry.objects.filter(
+            project=info.context.active_project,
+            **get_global_filters(_filter),
+        )
+        assessment_qs_filter = AssessmentDashboardFilterSet(queryset=assessment_qs, data=_filter.get("assessment")).qs
+        methodology_attribute_qs = MethodologyAttribute.objects.filter(assessment_registry__in=assessment_qs_filter)
+        cache_key = CacheHelper.generate_hash(_filter.__dict__)
+        return AssessmentDashboardStat(
+            cache_key=cache_key,
+            assessment_registry_qs=assessment_qs_filter,
+            methodology_attribute_qs=methodology_attribute_qs,
+        )
+
+    @staticmethod
+    def resolve_total_assessment(root: AssessmentDashboardStat, info) -> int:
+        return root.assessment_registry_qs.count()
+
+    @staticmethod
+    def resolve_total_stakeholder(root: AssessmentDashboardStat, info) -> int:
+        qs = root.assessment_registry_qs.values("stakeholders")
+        return qs.count()
```

Comment:
This gives total stakeholders count. But we need to return unique stakeholder count here.